Document pagination options for listing records (#1767)
zeit/docs: components/references-mdx/api/v2/endpoints/dns.mdx

| ------------- | ---------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------- |
| **id**        | <HelpLink href="#api-basics/types">ID</HelpLink>           | The unique ID of the DNS record. Always prepended with `rec_`.                                                  |
| **type**      | <HelpLink href="#api-basics/types">Enum</HelpLink>         | The type of record, it could be [any valid DNS record](https://en.wikipedia.org/wiki/List_of_DNS_record_types). |
| **name**      | <HelpLink href="#api-basics/types">String</HelpLink>       | A subdomain name or an empty string for the root domain.                                                        |
| **value**     | <HelpLink href="#api-basics/types">String</HelpLink>       | The record value.                                                                                               |
| **creator**   | <HelpLink href="#api-basics/types">String</HelpLink>       | The ID of the user who created the record or `system` if the record is an automatic record.                     |

> | **created**   | <HelpLink href="#api-basics/types">String</HelpLink>       | A timestamp with the date when the record was created.                                                          |

Reviewer: ```suggestion
| **created**   | <HelpLink href="#api-basics/types">Date</HelpLink>       | The date when the record was created.                                                          |
```

> | **updated**   | <HelpLink href="#api-basics/types">String</HelpLink>       | A timestamp with the date when the record was updated                                                           |

Reviewer: ```suggestion
| **updated**   | <HelpLink href="#api-basics/types">Date</HelpLink>       | The date when the record was updated.                                                           |
```

> | **createdAt** | <HelpLink href="#api-basics/types">String</HelpLink>       | A timestamp with the date when the record was created.                                                          |

Reviewer: ```suggestion
| **createdAt** | <HelpLink href="#api-basics/types">Date</HelpLink>       | The date when the record was created in milliseconds since the UNIX epoch.                                                          |
```

> | **updatedAt** | <HelpLink href="#api-basics/types">String</HelpLink>       | A timestamp with the date when the record was updated                                                           |

Reviewer: ```suggestion
| **updatedAt** | <HelpLink href="#api-basics/types">Date</HelpLink>       | The date when the record was updated in milliseconds since the UNIX epoch.                                                          |
```
